bundles/org.openhab.binding.mqtt.awtrixlight/README.md

```diff
@@ -0,0 +1,380 @@
+
+Rules DSL:
+
+```
+val awtrixActions = getActions("mqtt.awtrixlight", "mqtt:awtrixclock:myBroker:myAwtrix")
+```
+
+JS Scripting:
+
+```javascript
+var awtrixActions = actions.thingActions("mqtt.awtrixlight", "mqtt:awtrixclock:myBroker:myAwtrix");
+```
+
+### Indicator Control
+
+Control the three indicator LEDs on the Awtrix display (JS Scripting):
+
+```javascript
+// Blink indicator 1 in red for 1 second
+awtrixActions.blinkIndicator(1, [255,0,0], 1000)
+
+// Fade indicator 2 to blue over 2 seconds
+awtrixActions.fadeIndicator(2, [0,0,255], 2000)
+
+// Turn on indicator 3 in green
+awtrixActions.activateIndicator(3, [0,255,0])
+
+// Turn off indicator 1
+awtrixActions.deactivateIndicator(1)
+```
+
+### Device Control
+
+Control basic device functions:
+
+```javascript
+// Reboot the device
+awtrixActions.reboot()
+
+// Put device to sleep for 60 seconds
+awtrixActions.sleep(60)
+
+// Perform firmware upgrade
+awtrixActions.upgrade()
+```
+
+### Sound Control
+
+Play sounds and melodies:
+
+```javascript
+// Play a predefined sound file (without extension)
+awtrixActions.playSound("notification")
+
+// Play an RTTTL melody
+awtrixActions.playRtttl("Indiana:d=4,o=5,b=250:e,8p,8f,8g,8p,1c6,8p.,d,8p,8e,1f,p.,g,8p,8a,8b,8p,1f6,p,a,8p,8b,2c6,2d6,2e6,e,8p,8f,8g,8p,1c6,p,d6,8p,8e6,1f.6,g,8p,8g,e.6,8p,d6,8p,8g,e.6,8p,d6,8p,8g,f.6,8p,e6,8p,8d6,2c6")
+```
+
+### Notifications
+
+Display notifications on the screen:
+
+```
+// Show simple notification with icon
+awtrixActions.showNotification("Hello World", "alert")
+
+// Show custom notification with advanced options
+val params = newHashMap(
+    'text' -> 'Custom Message',
+    'icon' -> 'warning',
+    'color' -> [255,165,0],  // Orange color
+    'rainbow' -> true,
+    'duration' -> 10
+)
+awtrixActions.showCustomNotification(
+    params,     // Notification parameters
+    false,      // hold: Keep notification until manually cleared
+    true,       // wakeUp: Wake up from screen saver
+    true,       // stack: Add to notification stack
+    "Indiana:d=4,o=5,b=250:e,8p,8f,8g,8p,1c6,8p.,d,8p,8e,1f,p.,g,8p,8a,8b,8p,1f6,p,a,8p,8b,2c6,2d6,2e6,e,8p,8f,8g,8p,1c6,p,d6,8p,8e6,1f.6,g,8p,8g,e.6,8p,d6,8p,8g,e.6,8p,d6,8p,8g,f.6,8p,e6,8p,8d6,2c6", // RTTTL sound to play (not both sound and rtttl)
+    "alert",    // Sound file to play (not both sound and rtttl)
+    false       // loopSound: Loop the sound until manually stopped
+)
+```
+
+#### Custom Notification Parameters
+
+The action method parameters:
+
+| Parameter | Type | Description |
+|-----------|------|-------------|
+| `hold` | Boolean | Keep notification until manually cleared |
+| `wakeUp` | Boolean | Wake up from screen saver |
+| `stack` | Boolean | Add to notification stack |
+| `rtttl` | String | RTTTL melody to play |
+| `sound` | String | Sound file to play (without extension) |
+| `loopSound` | Boolean | Loop the sound |
+| `params` | Map | Notification parameters |
```

Comment:
Better to use markdowntable.com or any other markdown tool to align the table.

Reply:
Done